### ISSUE TITLE
SVG has colors changed on PDF2 render

### ISSUE THREAD

**Issue by cpkio** [dita-ot/dita-ot#3707]
## Expected Behavior

A color should be rendered in PDF2 as drawn in SVG.

## Actual Behavior

The colors are changed to #030303 color.

## Possible Solution

Render your map in DITA-OT 2.x. Not tested with any previous DITA-OT 3.x

## Steps to Reproduce

Download DITA-OT 3.6. Download the [dita-issue-test-kit.zip](https://github.com/dita-ot/dita-ot/files/6041468/dita-issue-test-kit.zip), unpack it, cd to `dita-issue-test-kit\managing-devices` and run `dita -i device-search-and-add.dita -f pdf`.

See the color of the image header in PDF. Compare it to original SVG in `dita-issue-test-kit\images` folder.

### Long story short

Original:
![Original](https://user-images.githubusercontent.com/1917113/109124054-8ad6e300-775b-11eb-9bc7-afb6222393f5.png)

Processed:
![2021-02-25_111947](https://user-images.githubusercontent.com/1917113/109124110-9aeec280-775b-11eb-9bd2-bb06c2fd0ccd.png)

## Copy of the error message, log file or stack trace

No errors generated.

## Environment

* DITA-OT version:
   _3.6_
* Operating system and version:
  _Windows_
* How did you run DITA-OT?
  `dita -i device-search-and-add.dita -f pdf`
* Transformation type:
  _PDF_
